lib/preprocess.js

```diff
@@ -248,6 +249,7 @@ export const assertValidMeasurement = measurement => {
       'stationId must be a hex string with 88 characters'
     )
   }
+  assert(!(measurement.head_status_code && !measurement.status_code), '`head_status_code` must have `status_code` as well')
```

Comment:
To be safe, we should also assert that `head_status_code` is a number. WDYT?

Reply:
We are currently not doing any type validation on measurements fields, eg I couldn't find any for `status_code` either. We're relying on `spark-publish` submitting this correctly (where we do have validation as well).

Reply:
ping @bajtos

Reply:
Fair enough 👍🏻